contracts/TreasuryVester.sol

```diff
@@ -47,16 +47,16 @@ contract TreasuryVester {
         require(block.timestamp >= vestingCliff, 'TreasuryVester::claim: not time yet');
         uint amount;
         if (block.timestamp >= vestingEnd) {
-            amount = IUni(uni).balanceOf(address(this));
+            amount = IVex(vex).balanceOf(address(this));
         } else {
             amount = vestingAmount.mul(block.timestamp - lastUpdate).div(vestingEnd - vestingBegin);
             lastUpdate = block.timestamp;
         }
-        IUni(uni).transfer(recipient, amount);
+        IVex(vex).transfer(recipient, amount);
     }
 }
 
-interface IUni {
+interface IVex {
```

Comment:
Another floor interface. Up to you if we move all of them to top of file or just leave them to preserve a minimal diff

Reply:
fixed